sdk/core/core-rest-pipeline/test/exponentialRetryPolicy.spec.ts

```diff
@@ -54,7 +54,7 @@ describe("exponentialRetryPolicy", function () {
 
     const clock = sinon.useFakeTimers();
 
-    const promise = policy.sendRequest(request, next);
+    const promise = assert.isRejected(policy.sendRequest(request, next), /Test Error/);
```

Comment:
Now we’re throwing the error than simply returning the response. This test change will prevent us from reversing it in the future.